sourcegraph/about
# Syntax Highlighting

Closes #5251 and adds syntax highlighting to code blocks in markdown.

### Changelog
- added prism rehype plugin to our markdown parsing lib
- added prism-themes and included the `prism-one-light` theme
- Updated next and mdx-remote
- removed old code element styles
- removed unused font families and fixed our font family declarations
- a few bug fixes in [fb0a31b](https://github.com/sourcegraph/about/pull/5520/commits/fb0a31b38515b9498b317bb1465ff4885d977aad) mentioned in [this comment](https://github.com/sourcegraph/about/pull/5520#issuecomment-1168965377)

### Test

1. Ensure prettier has standardized the proposed changes.
2. Check some post types for syntax highlighting (ie: `/blog/graphql/handling-authentication-and-authorization-in-graphql`)
